scripts/globals/items/caitiffs_socks.lua

```diff
@@ -0,0 +1,31 @@
+-----------------------------------
+-- ID: 15324
+-- Caitiff's Socks
+-- Players have chance to be given flee effect when taken damage
+-- while HP < 25 and TP < 1000
+-- Lasts for 30 seconds. Proc rate requires more information
+-----------------------------------
+local itemObject = {}
+
+itemObject.onItemDrop = function(player, item)
+end
+
+itemObject.onItemEquip = function(player, item)
+    player:addListener("ATTACKED", "CAITIFF_SOCKS_FLEE", function(playerArg, mob)
+        if
+            playerArg:getHPP() < 25 and
+            playerArg:getTP() < 1000 and
+            math.random() <= 15
```

Comment:
math.random() with no args produces a number between 0 and 1, which will always be true in this case.  I'm assuming you wanted
`math.random(100) <= 15`